source/command/atlas-kubernetes-dry-run.txt

```diff
@@ -0,0 +1,83 @@
+.. _atlas-kubernetes-dry-run:
+
+========================
+atlas kubernetes dry-run
+========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Deploy and run Atlas Kubernetes Operator in dry-run mode
+
+This command deploys the Atlas Kubernetes operator with the DryRun mode.
```

Comment:
Capitalize Operator
Should it be instead: "in the dry run mode", because this is just a description?